modules/cpmstarBidAdapter.js

```diff
@@ -0,0 +1,121 @@
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
```

Comment:
In lieu of this PR #3435, any imported files within a modules file needs to use the relative style path.

Can you please update these imports accordingly?

Reply:
We updated the imports.